src/setup.jl

```diff
@@ -171,13 +171,12 @@ end
 include(joinpath(dirname(@__FILE__),"..","deps","setup.jl"))
 
 function __init__()
-    if Rhome == ""
-        error(
+    # This should actually error much sooner, but this is just in case
+    isempty(Rhome) && error(
```

Comment:
Ha! Mixed feelings about this sneaky hack to pass coverage. The reason there is no coverage here is because when we shell out in `installation.jl` coverage stops being measured. It's probably not worth tracking it.

Reply:
Yeah, it's definitely a hack but it's one I'm comfortable with for such "defense in depth" type measures.